RFCs/0011-Discover-Only-TestProjects.md

```diff
@@ -0,0 +1,76 @@
+# 0011 Discover for Only Test Projects
+
+## Summary
+This note outlines the proposed changes to the TestWindow component inside VS IDE,
+for making improvements in the test discovery.
+
+## Motivation
+* Scope of improvement in discovery through TestExplorer. See [here](#Discovery.Proof)
+* Need to standardize the rules for identifying Test projects.
+
+## Problem
+TestWindow component inside VS IDE currently sends all the projects present in the solution for discovery.
+This list of projects includes projects that do not contain any tests.
+Sending redundant projects to the runner creates an overhead. Each adapter will try and fail to discover any tests inside such projects.
+
+### Scenarios
+There are two ways user can author C# based tests
+1. Create a test project using Unit Test template
+2. Add the appropriate adapter or framework references to a class library project. 
+
+### Current State
+There are existing ways to identify if a project is a test project. Brief explanation is as follows.
```

Comment:
nit: There are three mechanisms Test Explorer uses to identify test project. Number them below.